test/fops/file.c

```diff
@@ -159,10 +208,27 @@ obj_drop(int force)
 
 	testutil_check(conn->open_session(conn, NULL, NULL, &session));
 
+	if (use_txn)
+		testutil_check(session->begin_transaction(session, NULL));
 	if ((ret = session->drop(session, uri, force ? "force" : NULL)) != 0)
 		if (ret != ENOENT && ret != EBUSY)
 			testutil_die(ret, "session.drop");
 
+	if (use_txn) {
+		/*
+		 * As the operations are being performed concurrently,
+		 * return value can be ENOENT or EBUSY.
+		 * __wt_schema_drop when returns error, will set
```

Comment:
Again, don't refer to internal library functions in user programs.  This sentence can also just be removed and say that we must rollback the transaction.